examples/02-plot/lookup-table.py

```diff
@@ -0,0 +1,236 @@
+"""
+.. _lookup_table_example:
+
+Anti-Aliasing
+~~~~~~~~~~~~~
+Demonstrate the usage of a lookup table within PyVista
+
+The :class:`pyvista.LookupTable` can be used to have fine-tuned control over
+the mapping between a :class:`pyvista.DataSet`'s scalars and RGBA colors.
+
+"""
+
+import pyvista as pv
+from pyvista import examples
+
+pv.set_plot_theme('document')
+
+# download an example dataset
+bracket = examples.download_fea_bracket().cell_data_to_point_data()
+bracket
+
+
+###############################################################################
+# Default Color Map - Lookup Table
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# First, let's plot using the default color map, "viridis". Internally, PyVista
+# will automatically create a lookup table to map the scalars (stored here
+# within `point_data`, to RGBA colors. This is shown here as a nested attribute
+# to the :class:`pyvista.DataSetMapper` and it has a helpful ``repr`` method:
+
+pl = pv.Plotter()
+actor = pl.add_mesh(bracket)
+actor.mapper.lookup_table
+
+
+###############################################################################
+# Plot the Lookup Table
+# ~~~~~~~~~~~~~~~~~~~~~
+# You can also plot lookup table to see the mapping between the scalar values
+# (here, between 0 and 23.3) and RGBA colors.
+
+pl = pv.Plotter()
+actor = pl.add_mesh(bracket)
+actor.mapper.lookup_table.plot()
```

Comment:
I really don't like this logic... `Plotter` objects ought to be useable across cells. We should consider fixing this in the SG scraper

Reply:
While these get cleaned up automatically in sphinx gallery, they do not when a a user copy/pastes these snippets into their shell. I think this encourages improper use of the API and this is not intuitive

Reply:
Agreed and this should be fixed in a later PR.